harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -674,85 +706,95 @@ def _save(self, path: pathlib.Path) -> workload.Response:
+                models=[self.a, self.b],
+                optimizers=[self.opt1, self.opt2],
+                num_losses=2,
+            )
 
-        # TODO(DET-3204): Add the following comments to the docstring.
-        # Alternatively, we could initialize them with the methods provided by
-        # :py:class:`determined.pytorch.PyTorchTrialContext`, including ``Model``,
-        # ``Optimizer``, ``LRScheduler``, ``_configure_apex_amp``.
-        # Here is a code example.
-        #
-        # .. code-block:: python
-        #
-        #     self.context = context
-        #
-        #     self.a = self.context.Model(MyModelA())
-        #     self.b = self.context.Model(MyModelB())
-        #     self.opt1 = self.context.Optimizer(torch.optm.Adam(self.a))
-        #     self.opt2 = self.context.Optimizer(torch.optm.Adam(self.b))
-        #
-        #     (self.a, self.b), (self.opt1, self.opt2) = self.context._configure_apex_amp(
-        #         models=[self.a, self.b],
-        #         optimizers=[self.opt1, self.opt2],
-        #         num_losses=2,
-        #     )
-        #
-        #     self.lrs1 = context.LRScheduler(
-        #         lr_scheduler=LambdaLR(self.opt1, lr_lambda=lambda epoch: 0.95 ** epoch),
-        #         step_mode=LRScheduler.StepMode.STEP_EVERY_EPOCH,
-        #     ))
+            self.lrs1 = self.context.wrap_lrscheduler(
+                lr_scheduler=LambdaLR(self.opt1, lr_lambda=lambda epoch: 0.95 ** epoch),
+                step_mode=LRScheduler.StepMode.STEP_EVERY_EPOCH,
+            ))
+        """
         pass
 
     def build_model(self) -> nn.Module:
         """
-        Defines the deep learning architecture associated with a trial. This method
+        (Deprecated) Defines the deep learning architecture associated with a trial. This method
         returns the model as an instance or subclass of :py:class:`nn.Module`.
         """
-        # TODO(DET-3267): deprecate this when releasing pytorch flexible primitives.
+        # TODO(DET-3262): remove this backward compatibility of old interface.
         pass
 
     def optimizer(self, model: nn.Module) -> torch.optim.Optimizer:  # type: ignore
         """
-        Describes the optimizer to be used during training of the given model,
+        (Deprecated) Describes the optimizer to be used during training of the given model,
         an instance of :py:class:`torch.optim.Optimizer`.
         """
-        # TODO(DET-3267): deprecate this when releasing pytorch flexible primitives.
+        # TODO(DET-3262): remove this backward compatibility of old interface.
         pass
 
     def create_lr_scheduler(
         self, optimizer: torch.optim.Optimizer  # type: ignore
     ) -> Optional[LRScheduler]:
         """
-        Create a learning rate scheduler for the trial given an instance of the
+        (Deprecated) Create a learning rate scheduler for the trial given an instance of the
```

Comment:
There's still a bunch of these laying around, I think we should use the same `warning::` syantax as before.